app/org/sagebionetworks/bridge/dynamodb/DynamoStudy.java

```diff
@@ -39,6 +40,7 @@
     private String technicalEmail;
     private boolean usesCustomExportSchedule;
     private List<UploadFieldDefinition> uploadMetadataFieldDefinitions;
+    private UploadValidationStrictness uploadValidationStrictness;
```

Comment:
What happens to strictUploadValidationEnabled? Will it eventually go away?

Reply:
strictUploadValidationEnabled is deprecated in favor of the new Enum. I have no plans to remove it at the moment, and I'm not sure what migration will look like.

That said, in the Bridge Study Manager, we should be replacing strictUploadValidationEnabled with uploadValidationStrictnes.